src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,53 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+import { Tab } from '../../types/Tab';
+
+type TabsProps = {
+  tabs: Tab[],
+  selectedTabId: string,
+  onTabSelected: (selectedTab: Tab) => void,
+};
+
+export const Tabs: React.FC<TabsProps> = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  if (!tabs.find(tab => tab.id === selectedTabId)) {
+    // eslint-disable-next-line no-param-reassign
+    selectedTabId = tabs[0].id;
+  }
+
+  return (
+    <>
+      <div data-cy="TabsComponent">
+        <div className="tabs is-boxed">
+          <ul>
+            {tabs.map(tab => (
+              <li
+                className={classNames(
+                  { 'is-active': tab.id === selectedTabId },
+                )}
+                data-cy="Tab"
+                key={tab.id}
+              >
+                <a
+                  href={`#${tab.id}`}
+                  data-cy="TabLink"
+                  onClick={() => {
+                    return selectedTabId !== tab.id ? onTabSelected(tab) : null;
+                  }}
```

Comment:
DRY
Move this to the variable `selectedTabId !== tab.id`
```suggestion
                  onClick={() => {
                    if (!isTabSelected) {
                      onTabSelected(tab)
                    }
                  }}
```

Reply:
Done